src/components/app/app.scss

```diff
@@ -1,9 +1,9 @@
-@import "../../styles/variables";
+@import "../../styles/styles";
```

Comment:
I'm not a fan of that implementation. This also is repeated name.

What would you think of something like:

```diff
diff --git a/src/components/app/app.scss b/src/components/app/app.scss
index c7e1a68..6356af2 100644
--- a/src/components/app/app.scss
+++ b/src/components/app/app.scss
@@ -1,4 +1,4 @@
-@import "../../styles/styles";
+@import "../../styles/colors";
 
 .c-app {
   display: grid;
diff --git a/src/index.tsx b/src/index.tsx
index f8a7974..0078d03 100644
--- a/src/index.tsx
+++ b/src/index.tsx
@@ -2,6 +2,7 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 
 import { App } from './components/app';
+import './main.scss';
 
 ReactDOM.render(
   <App />,
diff --git a/src/main.scss b/src/main.scss
new file mode 100644
index 0000000..778c882
--- /dev/null
+++ b/src/main.scss
@@ -0,0 +1,3 @@
+@import "./styles/colors";
+@import "./styles/fonts";
+@import "./styles/global";
diff --git a/src/styles/styles.scss b/src/styles/styles.scss
deleted file mode 100644
index dc0fb9c..0000000
--- a/src/styles/styles.scss
+++ /dev/null
@@ -1,3 +0,0 @@
-@import "./fonts";
-@import "./colors";
-@import "./global";

```

You can copy the above and run `git apply <(pbpaste)` for it to change your files and see difference.